src/libraries/Microsoft.Extensions.Primitives/ref/Microsoft.Extensions.Primitives.cs

```diff
@@ -34,52 +37,48 @@ public partial interface IChangeToken
         bool HasChanged { get; }
         System.IDisposable RegisterChangeCallback(System.Action<object> callback, object state);
     }
-    [System.Diagnostics.DebuggerDisplayAttribute("Value = {_value}")]
-    [System.ObsoleteAttribute("This type is obsolete and will be removed in a future version.")]
```

Comment:
Obsolete needs to remain

Reply:
I can bring back the Obsolete, but this diff as-is was generated by GenerateReferenceSource target

Reply:
Understood.  Let me see about fixing that up by removing that suppression in a separate PR.